backend/python/tools/insert_test_data.py

```diff
@@ -174,7 +175,7 @@ def erase_db():
     db.session.commit()
     db.session.query(StoryTranslationAll).delete()
     db.session.commit()
-    db.session.query(User).delete()
```

Comment:
Only weird behaviour i observed goes as follows
- `python -m tools.insert_test_data`
- soft delete a user using this pr's mutation
- `python -m tools.insert_test_data erase`
- soft deleted users still remain in the database
- `python -m tools.insert_test_data`
- no users are double inserted, but soft deleted users are still deleted (ie is_deleted=1)
any thoughts on this @jennchenn ? We dont have to address this issue in this pr but it is a little odd. I tried doing each of the following locally but the same behaviour still occurred:
```
def erase_db():
    ...
    db.session.query(User).delete()
    db.session.query(UserAll).delete()
    ...
```
and 
```
def erase_db():
   db.drop_all()
```